frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -26,6 +26,7 @@ Cypress.Commands.add('install', (mode: 'Internal' | 'Attached' = 'Internal', enc
       cy.log('Subscribe to OCS Operator');
       cy.byLegacyTestID('operator-install-btn').click({ force: true });
       cy.byTestID('Operator recommended namespace:-radio-input').should('be.checked');
+      cy.byTestID('enable-monitoring').click();
       cy.byTestID('install-operator').click();
       cy.byTestID('success-icon', { timeout: 180000 }).should('be.visible');
       cy.exec('oc get project openshift-storage -o json').then((res) => {
```

Comment:
Up to this part, we can move to some global integration-tests-cypress folder, So anyone can reuse this installation logic by passing the operator name and namespace as param.

What I am suggesting here is, The operator install flow is a pre-requisite for OCS cluster creation flow. Instead of naming it as "install", we can name it something like prepate_storage_cluster or create_ocs_cluster. And prepare_ocs_cluster flow can the call install flow externally. For easier understanding separate these two install and create logic in different functions. (Or) Add "install" as  cypress global namespace from global cypress IT and Add "prepare_ocs_cluster" as cypress global namespace in ceph-storage-plugin cypress IT